egs/iwslt22_ta/ST/pruned_transducer_stateless5/train.py

```
@@ -0,0 +1,1290 @@
+
+                # NOTE: We use reduction==sum and loss is computed over utterances
+                # in the batch and there is no normalization to it so far.
+                if not inf_flag:
+                    scaler.scale(loss).backward()
+                    scheduler.step_batch(params.batch_idx_train)
+                    scaler.step(optimizer)
+                    scaler.update()
+                    optimizer.zero_grad()
+                else:
+                    continue
+            except:  # noqa
```

Ruff (0.12.2)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)